pulp_ansible/app/models.py

```diff
@@ -256,6 +257,16 @@ def download_factory(self):
             self._download_factory = AnsibleDownloaderFactory(self)
             return self._download_factory
 
+    @hook(
+        AFTER_UPDATE,
+        when_any=["url", "requirements_file", "sync_dependencies", "signed_only"],
+        has_changed=True,
+    )
+    def _reset_repository_last_synced_metadata_time(self):
```

Comment:
This approach does not seem to lock repositories, I believe this is problematic.

Reply:
I see at the bottom async_reserved_resources :+1: